tests/helpers.py

```diff
@@ -546,3 +546,17 @@ def create_predictions(
     ]
     dataset.upload_predictions(model, predictions)
     return predictions
+
+
+def assert_partial_equality(expected, result, base_keys=[]):
```

Comment:
can we add a comment that this is checking partial equality in that all the keys from expected must be present and matching in the result? was a little confusing to me at first glance before i figured that out

Reply:
like the direction of partial equality

Reply:
Added a comment.